docs/user-guide.md

```diff
@@ -28,9 +28,31 @@ Searching STAC Item can be filtered by:
 
 ![image](images/filters.png)
 
-### Add item as raster onto QGIS
+### Add item onto QGIS
+
+Result STAC Item can be added as raster and also vector ( the footprint) into QGIS.
```

Comment:
The plugin enables to loading STAC Item assets and footprints in QGIS as map layers.